sources/rest_api/config_setup.py

```diff
@@ -19,21 +19,22 @@
 from dlt.common.utils import update_dict_nested
 from dlt.common.typing import TSecretStrValue
 from dlt.sources.helpers.requests import Response
-
-from .auth import (
-    AuthConfigBase,
-    HttpBasicAuth,
-    BearerTokenAuth,
-    APIKeyAuth,
-    OAuthJWTAuth,
-)
-from .paginators import (
+from dlt.sources.helpers.rest_client.paginators import (
     BasePaginator,
     HeaderLinkPaginator,
     JSONResponsePaginator,
     SinglePagePaginator,
     JSONResponseCursorPaginator,
 )
+from dlt.sources.helpers.rest_client.exceptions import IgnoreResponseException
+from dlt.sources.helpers.rest_client.auth import AuthConfigBase
+from .auth import (
```

Comment:
ok then maybe we do not need auth module at all? we just import form core directly?